### ISSUE TITLE
Wrong value for GPIO_MODER_MODE8_Msk in all headers

### ISSUE THREAD

**Issue by kasjer** [STMicroelectronics/cmsis_device_f4#1]
**Describe the bug**
All headers for F4 board have this
```c
#define GPIO_MODER_MODE8_Msk             GPIO_MODER_MODER2_Msk
```
Changing **_8_** to **_2_** seems odd.
All surrounding lines have consistent numbering so it would make more sense to change it to
```c
#define GPIO_MODER_MODE8_Msk             GPIO_MODER_MODER8_Msk
```

**Comment by ASELSTM** [comment]
Hi @kasjer,

The issue you pointed out has been confirmed, a fix will be implemented and made available in future release. Thank you once again for your contribution.

With regards,

**Comment by ASELSTM** [comment]
ST Internal Reference: 86370

**Comment by ASELSTM** [comment]
Hi @kasjer,

I hope you are fine. The issue you reported has been fixed in the frame of version v2.6.6 of the cmsis_device_f4 published recently on GitHub.

Thank you again for having reported.

With regards,
